### PR TITLE
✨Source Appfollow: Migrate Python CDK to Low-code CDK

### LINKED ISSUE #29117: Appfollow

### PR DESCRIPTION
**Pull request by btkcodedev**
## What
*Migrating Source AppFollow to Low-Code CDK*
Closes #29117

## How
*Developed using (Configuration Based Source) low-code CDK*

## Recommended reading order
1. `spec.yaml`
2. `manifest.yaml`
3. `schemas/*`

## 🚨 User Impact 🚨
- No breaking changes, migration to low-code
<details><summary><strong>Migrate Connector: Source Appfollow</strong></summary>

### Community member or Airbyter

- [x] **Community member?** Grant edit access to maintainers ([instructions](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests))
- [x] Secrets in the connector's spec are annotated with `airbyte_secret`
- [x] Unit & integration tests added and passing. Community members, please provide proof of success locally e.g: screenshot or copy-paste unit, integration, and acceptance test output. To run acceptance tests for a Python connector, follow instructions in the README. For java connectors run `./gradlew :airbyte-integrations:connectors:<name>:integrationTest`.
- [x] Code reviews completed
- [x] Documentation updated
    - [x] Connector's `README.md`
    - [x] Connector's `bootstrap.md`
    - [x] `docs/integrations/<source or destination>/<name>.md` including changelog. See changelog [example](https://docs.airbyte.io/integrations/sources/stripe#changelog)
    - [x] `docs/integrations/README.md`
    - [x] `airbyte-integrations/builds.md`
- [x] PR name follows [PR naming conventions](https://docs.airbyte.com/contributing-to-airbyte/issues-and-pull-requests)

</details>

### CODE REVIEW

**Comment by btkcodedev** [comment]
Couldn't get the credentials, but has only one stream,
Reviewers could you please comment if changes are required?

File: airbyte-integrations/connectors/source-appfollow/source_appfollow/manifest.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+      type: "BasicHttpAuthenticator"
+      username: "{{ config['api_secret'] }}"
+      password: ""
+
+  retriever:
+    type: SimpleRetriever
+    record_selector:
+      $ref: "#/definitions/selector"
+    paginator:
+      type: NoPagination
```
Comment (archangelic) [inline airbyte-integrations/connectors/source-appfollow/source_appfollow/manifest.yaml:24]: Pagination needs to be configured. For example:

```
base_paginator:
    type: "DefaultPaginator"
    pagination_strategy:
      type: "CursorPagination"
      cursor_value: "{{ last_records['nextPageToken'] }}"
    page_token_option:
      type: "RequestPath"
      field_name: "pageToken"
      inject_into: "request_parameter"
```

See [the docs](https://docs.airbyte.com/connector-development/config-based/understanding-the-yaml-file/pagination) for more information.

File: airbyte-integrations/connectors/source-appfollow/metadata.yaml [hunk 2]
```diff
@@ -1,24 +1,25 @@
...
   githubIssueLabel: source-appfollow
   icon: appfollow.svg
   license: MIT
   name: Appfollow
-  registries:
-    cloud:
-      enabled: true
-    oss:
-      enabled: true
+  releaseDate: TODO
```
Comment (archangelic) [inline airbyte-integrations/connectors/source-appfollow/metadata.yaml:19]: A valid date in `YYYY-MM-DD` format needs to be set here. Today's date is fine.

**Review by archangelic — CHANGES_REQUESTED** [review]
Thanks so much for your contribution! I have listed some changes below.

**Review by marcosmarxm — APPROVED** [review]
Congrats @btkcodedev

**Review by archangelic — APPROVED** [review]

**Pull request merged**